### ISSUE TITLE
R / Rcpp

### ISSUE THREAD

**Issue by zachmayer** [FALCONN-LIB/FALCONN#52]
Seeing as you're planning a future Julia wrapper, I thought I'd ask for an R one too, so I can try FALCONN instead of [RcppAnnoy](https://github.com/eddelbuettel/rcppannoy).

**Comment by ludwigschmidt** [comment]
An R wrapper would definitely be nice to have. It's not on our immediate todo list right now but we'll probably look into it in the future.

Is Rcpp the preferred way for exposing C++ libraries to R?

**Comment by dselivanov** [comment]
@ludwigschmidt  it is much simpler to expose any C++ library to R with Rcpp than native R's C interface. So I can say it is preferred way.

**Comment by zachmayer** [comment]
Yeah, Rcpp is the preferred way to expose C++ libraries in R.  You might also be able to get someone who already knows Rcpp to do a lot of the work for you =D
